homeassistant/components/mqtt/cover.py

```diff
@@ -247,11 +247,8 @@ def __init__(
         discovery_data: DiscoveryInfoType | None,
     ) -> None:
         """Initialize the cover."""
-        self._position: int | None = None
-        self._state: str | None = None
-
+        self._attr_is_closed = None
         self._optimistic: bool | None = None
-        self._tilt_value: int | None = None
         self._tilt_optimistic: bool | None = None
```

Comment:
Can these move outside of the constructor? Then you can remove the whole constructor?

Reply:
We need to keep _attr_is_closed here as it is not set in CoverEntity, and it needs initialization only once.